packages/cloud-cognitive/src/components/Datagrid/styles/_draggableElement.scss

```diff
@@ -56,3 +72,19 @@ svg.#{$block-class}__draggable-handleStyle.disable {
   text-transform: none;
   white-space: nowrap;
 }
+
+[selected='true'] {
+  background-color: $layer-selected;
+}
+
+[selected='true']:hover {
+  background-color: $hover-selected-ui;
+}
+
+.-selected {
+  background-color: $layer-selected;
+}
+
+.-selected:hover {
+  background-color: $hover-selected-ui;
+}
```

Comment:
These all need to be scoped within the block class.

```scss
.#{$block-class} [selected='true'] {
...
}
```

Reply:
Reconsolidated selectors. These selectors have been removed to reduce redundancy and improve readability.